entries/s/sjsu.edu.json

```diff
@@ -12,7 +11,10 @@
     "custom-software": [
       "Duo"
     ],
-    "documentation": "https://www.sjsu.edu/it/support/service-desk/duo/index.php",
+    "custom-hardware": [
+      "Duo D100 Token"
```

Comment:
```suggestion
      "Feitian OTP c100 Token"
```

The following image is from the documentation mentioned in the initial comment.

![Feitian OTP c100 Token](https://kb.mlml.sjsu.edu/uploads/images/gallery/2019-04-Apr/scaled-840-0/Duo-Fob-OTP.jpg)